index.bs

```diff
@@ -235,8 +252,26 @@ To <dfn>sanitize an attribute</dfn> named |attr|, run these steps:
 6. if |config| has a non-empty [=attribute allow list=] and |name| is not in |config|'s [=attribute allow list=] return 'drop'
 7. return 'keep'
 
+To <dfn>create a document fragment</dfn>
+named |fragment| from a Sanitizer |input|, run these steps:
+
+1. Switch based on |input|'s type:
+  1. if |input| is of type {{DocumentFragment}}, return |input|.
```

Comment:
I think we need to create a new `DocumentFragment`, in order not to copy potentially assigned observers / event listeners

Reply:
Done.

Reply:
We're still using the input fragment here, aren't we?
Rest looks good btw

Reply:
I tried to fix that in the 3rd commit, which always to |node|, and then always does a cloneNode of |node| before returning it.

(Should I squash those commits before review? I'm still a little confused about the workflow here.)

Reply:
Actuallly.... :)  my version is broken. I still run cloneNode on |input|, rather than |node|. Let me fix that...

Reply:
Done.